backend/tests/unit/controllers/posts-controller.test.js

```diff
@@ -67,10 +67,27 @@ describe('createPostHandler', () => {
 
     expect(res.status).toHaveBeenCalledWith(400);
     expect(res.json).toHaveBeenCalledWith({
-      message: 'Image URL must end with .jpg, .jpeg, or .png',
+      message: 'Image URL must end with .jpg, .jpeg, .webp, or .png',
     });
   });
 
+  it('Post creation: Success - Valid Image URL (.webp format)', async () => {
+    const postObject = createPostObject({
```

Comment:
Thank you for your handy, test case, but can we revert this, and instead place 
`imageLink: options.imageLink || 'https://www.forTestingPurposeOnly/my-image.jpg',` <-- Let's change it here to .webp
Let's stick the existing test cases, because there are many test cases and it becomes very difficult to track these.